### PR TITLE
[#10664] Student trying to access non-existent session: give a better error message 

### LINKED ISSUE #10664: Student trying to access non-existent session: give a better error message
Current:
image

Suggestion: 
* Give a better error message e.g., `The session does not exist (most likely has been deleted by the instructor after the submission link was sent out)`. 
* Not show the partially loaded page?

### PR DESCRIPTION
**Pull request by sjkadali**
Fixes #10664

**PR Checklist**

Ensure that you have:
- [ y] Read and understood our PR guideline: https://github.com/TEAMMATES/teammates/blob/master/docs/process.md#step-4-submit-a-pr
  - [ y] Added the issue number to the "Fixes" keyword above
  - [y ] Titled the PR as specified in the abovementioned document
- [y] Made your changes on a branch other than `master` and `release`
- [y ] Gone through all the changes in this PR and ensured that:
  - [ ] They addressed one (and only one) issue
  - [ y] No unintended changes were made
- [ ] Run and passed static analysis: `./gradlew lint` and `npm run lint`
- [ ] Added/updated tests, if changes in functionality were involved
- [ ] Added/updated documentation to public APIs (classes, methods, variables), if applicable

**Outline of Solution**

### CODE REVIEW

**Comment by ChooJeremy** [comment]
Hi @sjkadali, have you tested this locally? Does the error message appear as expected?

**Comment by sjkadali** [comment]
@ChooJeremy , when I tested this locally, it showed the below page with error message.

Screenshot from 2020-10-02 10-45-33

File: src/web/app/pages-session/session-submission-page/session-submission-page.component.ts [hunk 1]
```diff
@@ -314,10 +314,11 @@ export class SessionSubmissionPageComponent implements OnInit, AfterViewInit {
         this.loadFeedbackQuestions();
       }, (resp: ErrorMessageOutput) => {
         if (resp.status === 404) {
-          this.simpleModalService.openInformationModal('Feedback Session Deleted!', SimpleModalType.DANGER,
-            'The feedback session has been permanently deleted and is no longer accessible.');
+          this.simpleModalService.openInformationModal('Feedback Session Does Not Exist!', SimpleModalType.DANGER,
+            'The session does not exist (most likely deleted by instructor after the submission link was sent).');
```
Comment (damithc) [inline src/web/app/pages-session/session-submission-page/session-submission-page.component.ts:318]: Change as `by the instructor`

**Review by damithc — CHANGES_REQUESTED** [review]

**Comment by ChooJeremy** [comment]
> 
> 
> Hi @sjkadali, have you tested this locally? Does the error message appear as expected?

That's odd, when I tested this the redirect occurs immediately (no chance for an error message to appear), how are you testing this exactly?

File: src/web/app/pages-session/session-submission-page/session-submission-page.component.ts (hunk 1 above)
Comment (sjkadali) [reply src/web/app/pages-session/session-submission-page/session-submission-page.component.ts:318]: ok

**Comment by sjkadali** [comment]
> > Hi @sjkadali, have you tested this locally? Does the error message appear as expected?
> 
> That's odd, when I tested this the redirect occurs immediately (no chance for an error message to appear), how are you testing this exactly?

@ChooJeremy, i had started the dev server in the background using the command,  ./gradlew appengineRun to access the admin page at http://localhost:8080/web/admin/home.  Then started Front-end dev server using the command, npm run start.  
On updating the error message, logged in as student at http://localhost:4200/web/student/home. Clicked on an existing submission's View Submission button to view the submission. On changing the link to point to a non-existing session,  it redirects to the student home page , but the updated error message is shown.

**Comment by sjkadali** [comment]
@damithc , Have addressed the requested changes. Please review. Thanks!

**Review by ChooJeremy — APPROVED** [review]
Ah, I see. Nice, so this solves the issue of the partially loaded page as well. LGTM

**Review by damithc — APPROVED** [review]
The message is OK.

**Pull request merged**